mantis-runtime/src/main/java/io/mantisrx/runtime/executor/WorkerPublisherRemoteObservable.java

```diff
@@ -42,31 +42,33 @@
 import rx.Observable;
 import rx.functions.Func1;
 
-
-public class WorkerPublisherRemoteObservable<T, R> implements WorkerPublisher<T, R> {
+/**
+ * Execution of WorkerPublisher that publishes the stream to the next stage.
+ *
+ * @param <T> incoming codec
+ */
+public class WorkerPublisherRemoteObservable<T> implements WorkerPublisher<T> {
 
     private static final Logger logger = LoggerFactory.getLogger(WorkerPublisherRemoteObservable.class);
 
-    private String name;
-    private int serverPort;
+    private final String name;
+    private final int serverPort;
     private RemoteRxServer server;
-    private Observable<Integer> minConnectionsToSubscribe;
-    private MantisPropertiesService propService;
+    private final MantisPropertiesService propService;
     private String jobName;
 
     public WorkerPublisherRemoteObservable(int serverPort,
                                            String name, Observable<Integer> minConnectionsToSubscribe,
                                            String jobName) {
         this.name = name;
         this.serverPort = serverPort;
-        this.minConnectionsToSubscribe = minConnectionsToSubscribe;
         this.propService = ServiceRegistry.INSTANCE.getPropertiesService();
         this.jobName = jobName;
     }
 
     @SuppressWarnings( {"rawtypes", "unchecked"})
     @Override
-    public void start(final StageConfig<T, R> stage, Observable<Observable<R>> toServe) {
+    public void start(final StageConfig<?, T> stage, Observable<Observable<T>> toServe) {
```

Comment:
This is an interesting change, wasn't expecting it. 
LGTM but would appreciate a closer look from one more person here. perhaps @codyrioux or @Andyz26

Reply:
I just took a look through and this appears to be fine. I like that it is more explicit about the fact that we are not using the first type parameter in the `StageConfig` in these locations.